### ISSUE TITLE
Extend song format to allow padding

### ISSUE THREAD

**Issue by NoctuaNivalis** [vimus/vimus#77]
I think it would add a lot of readability to the song lists if they were shown in columns. This could easily be achieved by allowing some notation for padding/truncating overflow in the song format language.

Even cooler (but guessing from some looks at the code, requiring a lot more changes) would be if this padding (column width) could be relative to the window width.

**Comment by sol** [comment]
Yes, I'd love to have aligned columns.  If you can come up with a patch, that would be more than welcome.

**Comment by sol** [comment]
@NoctuaNivalis sorry for the late reply btw.
